slack-api-model-kotlin-extension/src/main/kotlin/com/slack/api/model/kotlin_extension/block/element/ButtonStyle.kt

```diff
@@ -6,16 +6,16 @@ enum class ButtonStyle {
      * only be used for one button within a set.
      */
     PRIMARY {
-        override val value: String? = "primary"
+        override val value = "primary"
     },
 
     /**
      * Danger gives buttons a red outline and text, and should be used when the action is destructive. Use danger even
      * more sparingly than primary.
      */
     DANGER {
-        override val value: String? = "danger"
+        override val value = "danger"
     };
 
-    abstract val value: String?
+    abstract val value: String
```

Comment:
Good catch

Reply:
But also reverted in 1be897c8ce41fb2d32ee1b4cdaa0d647fd7b43c0 😅